search-app/src/cmr/search/data/query_to_elastic.clj

```diff
@@ -277,6 +281,10 @@
   [_]
   {:tag-key :tag-key.lowercase})
 
+(defmethod q2e/concept-type->sort-key-map :variable
+  [_]
+  {:variable-name :variable-name.lowercase})
```

Comment:
I think it's just a carryover from catalog-rest - don't exactly know the history. It turns out that later versions of Elasticsearch do not allow dots in field names:

https://discuss.elastic.co/t/field-name-cannot-contain/33251

So we'll have to deal with that if we update to Elasticsearch 2.0 or later. Since we are already using this convention throughout and we have functions that expect lowercase fields to end with .lowercase I'd say keep using dots for now and we'll deal with changing the field names when we upgrade.